flocker/acceptance/node_scripts/disable_service.py

```diff
@@ -0,0 +1,20 @@
+"""
+Stop a service and disable across reboots.
+"""
+
+import os
+import sys
+from subprocess import check_call
+
+service = sys.argv[1]
+
+if os.path.exists("/etc/redhat-release"):
+    # Redhat-based system:
+    check_call(["systemctl", "disable", service])
+    check_call(["systemctl", "stop", service])
+else:
+    # Ubuntu 14.04
+    override = "/etc/init/%.override" % (service,)
+    with file(override, "w") as f:
+        f.write("manual\n")
+    check_call(["service", service, "start"])
```

Comment:
I'm guessing this is supposed to be "stop" instead of "start" - probably add a comment as to why this isn't naively just the ubuntu version of the redhad commands if I'm wrong.